working-groups/user-interface/About.md

```diff
@@ -0,0 +1,44 @@
+# User-interface (UI) working group
+Hello and welcome. This page talks about the UI working group. If you have any queries about the UI working group after going through this page please feel free to reach out to us on [slack.](https://join.slack.com/t/mojaglobal/shared_invite/zt-o6ta1ug0-rVLjAo460~d7JbZ~HpFFtw)
+
+## What’s a working group?
+A working group is a group of people working together to achieve specified goals. The groups are domain-specific and focus on discussion or activity around a specific subject area.
+
+## About the UI working group
+<a href="#incubating"><img src="https://img.shields.io/static/v1?label=Status &message=Active &color=green" /></a>
+
+The UI working group is responsible for developing the visual components, user interactions and user experience of the UI-based projects under moja global.
+
+| Meeting details            |                                                                                                    |
+| -------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| Leads                      | [@Shloka Gupta](https://github.com/chicken-biryani?tab=followers) [@Soham Shah](https://github.com/sohamsshah) |
```

Comment:
Why does your URL lead to individual followers list and not to your direct profile?